docs/functions/Bundles/Start-UPloadVCFBundle.md

```diff
@@ -0,0 +1,38 @@
+# Start-UploadVCFBundle
```

Comment:
Function name is not consistent with the module. Suggest Start-VCFBundleUpload.

Reply:
Fixed